### DISCUSSION TITLE
New translation in Amharic

### DISCUSSION THREAD

**Discussion by javannnn** [sircharlo/meeting-media-manager#2551] [Translations]
### What language do you speak?

I would like to help translate M³ into a language I speak, Amharic (AM/am).

**Comment by sircharlo** [comment]
Thank you so much for your willing spirit!
If you would like to volunteer to become a M³ language moderator for a language you speak, please [sign into Crowdin](https://crowdin.com/project/meeting-media-manager) using your GitHub account, and then let me know once that is done.

As language moderator, you will get notified of changes to the English original strings, have a nice and easy to use interface to review existing as well as untranslated strings in your language, and submit new translations to the repo easily.

**Comment by javannnn** [comment]
Hi Charles,

Thank you for the opportunity!

I have signed into Crowdin using my GitHub account as requested. I am
excited to contribute as a language moderator and help with the translation
process. Please let me know the next steps or any additional information
you might need from me.

Looking forward to getting started!

Best regards,
Yafet

**Reply by sircharlo** [reply]
Awesome! Please note, within the next few weeks/months, we will release a brand new version of the app, so I would hold off until that version is out to prevent duplicate work.

**Comment by javannnn** [comment]
Sounds good, thanks for the heads up!

**Reply by sircharlo** [reply]
Thanks again for joining CrowdIn! I've added you as a moderator for Amharic. The file that contains the strings to translate for the new app is in the folder called `new-strings`.

**Comment by javannnn** [comment]
Hello Charles,

I'm getting this dialogue box when clicking on join or trying to open the
new-strings folder.
[image: image.png]

**Reply by mtdvlpr** [reply]
Hi @javannnn the image you tried to include is not included. Could you try again?

**Comment by javannnn** [comment]
Sorry let me try again here is the image

**Comment by mtdvlpr** [comment]
@javannnn, can you try opening GitHub in the browser instead of via email?

**Comment by javannnn** [comment]
Hi Manoah, Unfortunately I've signed in using github and I get the same
message "Join Translation Team
Your request has been submitted for approval.
If you would not receive a reply within 7 days, please contact a project
manager"

**Comment by mtdvlpr** [comment]
Try this link:

https://crowdin.com/project/meeting-media-manager/invite?h=c570a659d96138a5b9bf4a6a821594a92244681

**Comment by javannnn** [comment]
Tried it, no luck.

**Reply by mtdvlpr** [reply]
What does it say?

**Reply by mtdvlpr** [reply]
Could you try deleting your account and sign up again using GitHub? When you've done that, use the below link to join.

Delete account: https://crowdin.com/settings#account

Join our project: https://crowdin.com/project/meeting-media-manager/invite?h=f5b938d554b2fc946f6501cabc65b6582244719

**Comment by mtdvlpr** [comment]
Any luck, @javannnn?

**Reply by javannnn** [reply]
I deleted my account and joined using my GitHub account, after that I've been asked to submit an answer as to why I want to join the translation team, I've answered that  and submitted my request, now it's telling me request has been submitted for approval.

**Reply by javannnn** [reply]
![image](https://github.com/user-attachments/assets/c4c36eb0-0008-476b-9366-182dc97189a8)

**Comment by mtdvlpr** [comment]
@javannnn, I've received your request and accepted it! You should be good to go now.

**Reply by mtdvlpr** [reply]
Let me know if this is not the case.

**Reply by javannnn** [reply]
Awesome I'm all set! Thanks for the help.
